### ISSUE TITLE
Integrated Console and Transcript shows extra stuff

### ISSUE THREAD

**Issue by dbaduck** [PowerShell/vscode-powershell#1265]
### System Details

- Operating system name and version: Windows 10 (10.0.16299.251)
- VS Code version: 
Version 1.22.1
Commit 950b8b0d37a9b7061b6f0d291837ccc4015f5ecd
Date 2018-04-06T02:26:57.615Z
Shell 1.7.12
Renderer 58.0.3029.110
Node 7.9.0
Architecture x64
- PowerShell extension version: 
- Output from `$PSVersionTable`:
Name                           Value
----                           -----
PSVersion                      5.1.16299.251
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.16299.251
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```
Copy / paste the following commands into the PowerShell Integrated Console, and paste the output here:

code -v
PS C:\Users> code -v
1.22.1
950b8b0d37a9b7061b6f0d291837ccc4015f5ecd
x64
$pseditor.EditorServicesVersion
Major  Minor  Build  Revision
-----  -----  -----  --------
1      6      0      0

code --list-extensions --show-versions
ms-vscode.PowerShell@1.6.0

$PSVersionTable
Name                           Value
----                           -----
PSVersion                      5.1.16299.251
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.16299.251
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

I am experiencing a problem with transcripts in Integrated Console. When I use the Integrated Console with Transcript, I get messages like the ones below. This content throughout my transcript makes it very hard to distribute a transcript. When I use a second terminal PowerShell window it does not exhibit these problems, only the Integrated ones.  I demo this a lot and I would still like to use Code.
```
Transcript started, output file is .\documents\mytranscript_new.txt
PS>@{ 'computerName' = if ([Environment]::MachineName) {[Environment]::MachineName}  else {'localhost'}; 'processId' = $PID; 'instanceId' = $host.InstanceId }

Name                           Value
----                           -----
processId                      1312
computerName                   MACHINE
instanceId                     aa0eeae3-3de6-4361-b907-fecca790f081

PS C:\Demos\SmoStudy> @{ 'computerName' = $env:ComputerName; 'processId' = $PID; 'instanceId' = $host.InstanceId }

Name                           Value
----                           -----
processId                      1048
computerName                   SQLDEMO1
instanceId                     7948c207-c74a-46db-8ccb-93823065b319


PS C:\Demos\SmoStudy> CommandInvocation(Get-History): "Get-History"

  Id CommandLine
  -- -----------
   1 start-transcript -Path c:\demos\precon_madison.txt
   2 get-command -module sqlserver
   3 $env:psmodulepath -split ';'
```
I can generate logs if needed.

### Attached Logs

Follow the instructions in the [README](https://github.com/PowerShell/vscode-powershell#reporting-problems)
about capturing and sending logs.

**Comment by dbaduck** [comment]
Any updates on this one?  It is still there and it makes it hard to use Code and Azure Data Studio as demo places for PowerShell. Transcripts are key for learning and remembering after the time is done.

**Comment by JustinGrote** [comment]
Related to #873

**Comment by TheOnlyWei** [comment]
I have the same issue. Wasted a few hours debugging this futilely, then realized might be an issue with VS Code extension itself. This should be fixed just due to the fact that this issue is difficult to debug and you will waste a few hours on it until you realize it might just be an issue with the VS Code extension and nothing to do with your code.

The workaround to see your actual Start-Transcript output without VS Code extension clutter is obviously just to call the script you want to execute without running the script through the VS Code debugger.

**Comment by andschwa** [comment]
@TheOnlyWei @dbaduck Hi there, we just released [v2022.4.0-preview](https://github.com/PowerShell/vscode-powershell/releases/tag/v2022.4.0-preview)! Could you please try [PowerShell Preview for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell-Preview) and verify this is fixed? It seemed to be in my testing, so marking as resolved, but please let me know if it's still occurs (or if new bugs show up)!
